tests/bluetooth/tester/prj.conf

```diff
@@ -22,6 +22,8 @@ CONFIG_BT_DEVICE_NAME_GATT_WRITABLE=y
 
 CONFIG_BT_MESH=y
 CONFIG_BT_MESH_RELAY=y
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_MESH_ADV_EXT=y
```

Comment:
It isn't required to be set manually. It is done by KConfig system by choice: https://github.com/nrfconnect/sdk-zephyr/blob/main/subsys/bluetooth/mesh/Kconfig#L360